docs/examples/gen_velocities.rst

```diff
@@ -0,0 +1,432 @@
+                                \sin( \theta_\mathrm{p} )
+                         \right].
+
+Here, :math:`\theta_\mathrm{p}( t )` is the phase of pulsar orbit as measured
+from its ascending node.
+
+.. plot::
+    :include-source:
+    :context:
+
+    th_p = ((t - t0_p) / p_b).to('') * u.cycle
+
+    v_p_orb = (-k_p / np.sin(i_p)
+               * (np.cos(i_p) * np.sin(delta_omega_p) * np.cos(th_p)
+                              - np.cos(delta_omega_p) * np.sin(th_p)))
+
+Pulsar system barycentre velocity
+---------------------------------
+
+The pulsar system barycentre velocity projected onto the screen is given by
+
+.. math::
+
+    v_\mathrm{p,pm} \simeq d_\mathrm{p}
+                           \left[ \mu_\alpha \sin( \mathit{\Omega}_\mathrm{s} )
+                                + \mu_\delta \cos( \mathit{\Omega}_\mathrm{s} )
+                           \right].
+
+This can be computed manually, but it can also be retrieved from the
+:py:class:`~astropy.coordinates.SkyCoord` of the pulsar system (which contains
+the system's proper motion) by transforming it to ``lens_frame``.
+
+.. plot::
+    :include-source:
+    :context:
+    
+    v_p_pm = psr_coord.transform_to(lens_frame).velocity.d_z
+
+Effective velocity
+------------------
+
+Combine the velocities of the pulsar, Earth, and the lens into the effective
+velocity
+
+.. math::
+
+    v_\mathrm{eff} = \frac{1}{s} v_\mathrm{ISM}
+                     - \frac{1 - s}{s} ( v_\mathrm{p,orb} + v_\mathrm{p,pm} )
+                     - v_\mathrm{E}
+
+.. plot::
+    :include-source:
+    :context:
+    
+    v_eff = 1. / s * v_ism - ((1 - s) / s) * (v_p_orb + v_p_pm) - v_earth
+
+
+Curvature and scaled effective velocity
+=======================================
+
+The curvature :math:`\eta` can be computed from the effective velocity
+according to
+
+.. math::
+    
+    \eta = \frac{ \lambda^2 d_\mathrm{eff} }{ 2 c v_\mathrm{eff}^2 },
+
+where :math:`\lambda` is the observing wavelength and :math:`c` is the speed of
+light.
+
+.. plot::
+    :include-source:
+    :context:
+
+    lambda_obs = (1400. * u.MHz).to(u.m, equivalencies=u.spectral())
+
+    eta = lambda_obs**2 * d_eff / (2. * const.c * v_eff**2)
+
+Have a look at the curvature at a function of time.
+
+.. plot::
+    :include-source:
+    :context:
+
+    plt.figure(figsize=(12., 6.))
+    
+    quantity_support()
+    time_support(format='iso')
+    
+    plt.plot(t, eta.to(u.s**3))
+
+    plt.ylabel(r'curvature $\eta$ (s$^3$)')
+    
+    plt.show()
+
+Since :math:`v_\mathrm{eff}` can be arbitrarily close to zero (letting
+:math:`\eta` blow up), curvature has a strongly non-uniform prior probability
+distribution (as can be seen from the modulation in amplitude in the figure
+above). For this reason, it is sometimes better to fit for the curvature of the
+secondary spectrum parabola in a space of "scaled effective velocity"
+
+.. math::
+    
+    \frac{ \lambda }{ \sqrt{ 2 \eta c } }
```

Comment:
Maybe give this quantity its own symbol

Reply:
In this particular case, of course we actually know `d_eff`... But I think it is good to write this as a quantity purely derived from observations. I wonder if it makes sense to introduce a new term here other than curvature (which from wikipedia is actually *not* just "a" for `y=a x**2`. Instead, one might go for focal length or semi-latus rectum... I think it actually makes things easier if we go for something like `1/a`...

Reply:
Bottom line: let's ask tomorrow about what quantity one should really use!

Reply:
I left it as it was for now.